### PR TITLE
fear(agenda): add data-cy in delete scenario

### PR DESCRIPTION
**Pull request by fauziMK**
### Related
https://app.clickup.com/t/860rqp23m

### Overview
add data-cy in delete form

### Changes
add data-cy on delete scenario

### Evidence
project: Portal Jabar
title: add data attribut on form detail agenda
participants: @fauziMK @wulanLastia @farrasnaufalwork
lampiran: https://i.ibb.co/v4Q2bXB/Screenshot-2023-09-13-at-16-13-06.png

### CODE REVIEW

File: cypress/e2e/agenda/detail/positive.cy.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ beforeEach(() => {
 
 describe('Detail Agenda Scenario', () => {
     qase(255,
-        it('S27A2 - Detail Online - View Detail Agenda', () => {
+        it.only('S27A2 - Detail Online - View Detail Agenda', () => {
```
Comment (wulanLastia) [inline cypress/e2e/agenda/detail/positive.cy.js:46]: punten zi kalo yang ini memang di set .only?

Comment (fauziMK) [reply cypress/e2e/agenda/detail/positive.cy.js]: done teh

**Review by farrasnaufalwork — APPROVED** [review]
Dari aku sudah aman ya kang

**Review by wulanLastia — APPROVED** [review]
oke aman

**Pull request merged**